bedevere/bpo.py

```diff
@@ -37,6 +51,14 @@
         else:
             status = FAILURE_STATUS
     else:
+        if "body" in event.data["pull_request"]:
```

Comment:
Yes, there are edits (iirc) which does not have body, my test instance crashed due to this.